docs/endpoints/openrtb2/auction.md

```diff
@@ -125,17 +125,18 @@ to set these params on the response at `response.seatbid[i].bid[j].ext.prebid.ta
 
 ```
 {
-  "pricegranularity": [{
-     "precision": 2,
-     "min":0.0,
-     "max":20.0
-     "increment":0.1 }], // Required property.
+  "pricegranularity": {
+       "precision": 2,
+           "ranges": [{
+           "max":20.0
```

Comment:
Json snippet should be corrected. A comma is missed after _max_ attribute value and indentation is not perfect.